bin/rbenv-each

```diff
@@ -15,20 +18,42 @@ set -e
 # Provide rbenv completions
 if [ "$1" = "--complete" ]; then
   echo -v
+  echo --verbose
+  echo -a
+  echo --aliases
   exit
 fi
 
-if [ "$1" = "-v" ]; then
-  verbose=1
-  shift
-fi
+usage() {
+    rbenv-help --usage each >&2
+    exit 1
+}
 
-case "$1" in
-""  | -* )
-  rbenv-help --usage each >&2
-  exit 1
-  ;;
-esac
+verbose=
+aliases=
+
+while [[ $1 == -* ]]; do
+    case "$1" in
+        -v|--verbose)
+            verbose=1
+            ;;
+        -a|--aliases)
```

Comment:
Could we rename the option to `--with-aliases` so its name more closely suggests what it does?